vars/runATH.groovy

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env groovy
+
+/**
+ * Simple wrapper for running the ATH
+ */
+
+def call(Map params = [:]) {
+
+    def String athUrl = params.get('athUrl', 'https://github.com/jenkinsci/acceptance-test-harness.git')
+    def String athRevision = params.get('athRevision', 'master')
+    def String metadataFile = params.get('metadataFile', 'essentials.yml')
+    def String jenkins = params.get('jenkins', 'latest')
+
+    def isLocalATH = athUrl.startsWith("file://")
+    def isVersionNumber = (jenkins =~ /^(\d+\.)?(\d+\.)?(\*|\d+)$/).matches()
+    // Workaround for https://issues.jenkins-ci.org/browse/JENKINS-27092
+    def shouldStop = false
+    // End of workaround
+
+    def mirror = "http://mirrors.jenkins.io/"
```

Comment:
Please do not use the mirror sites to download artifacts during CI builds. This will strain our servers.

Better to obtain `jenkins.war` via `mvn dependency:get`. There may or may not be any local caching happening (probably each build on ci.jenkins.io gets a fresh local repo) but at least we can make it go through

```xml
<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">
    <mirrors>
        <mirror>
            <id>azure</id>
            <url>https://repo.azure.jenkins.io/public/</url> <!-- INFRA-1176 -->
            <mirrorOf>repo.jenkins-ci.org</mirrorOf>
        </mirror>
    </mirrors>
</settings>
```